notebooks/Chapter09_Random_Walk.jl

```diff
@@ -253,31 +253,6 @@ begin
 	(pp::PolynomialPreprocessor)(s::Number) = [s^i for i = 0:pp.order]
 end
 
-# ╔═╡ 08d133a0-5c77-11eb-1fbb-ed6b8da42d9f
-md"""
-## Figure 9.10
-
-Implementing the tile encoding in Julia is quite easy！😀
-"""
-
-# ╔═╡ 2ef2aa46-5c77-11eb-1eec-13ad13061214
-begin
-	struct Tiling{N,Tr<:AbstractRange}
-		ranges::NTuple{N,Tr}
-		inds::LinearIndices{N,NTuple{N,Base.OneTo{Int}}}
-	end
-
-	Tiling(ranges...) =Tiling(
-		ranges,
-		LinearIndices(Tuple(length(r) - 1 for r in ranges))
-	)
-
-	Base.length(t::Tiling) = reduce(*, (length(r) - 1 for r in t.ranges))
-end
-
-
-# ╔═╡ 7ee0867c-5c6d-11eb-11b4-a7858177564f
-NA = length(ACTIONS)
```

Comment:
Oops, it seems this line is removed by accident?

See the error message here:

https://juliareinforcementlearning.org/ReinforcementLearningAnIntroduction.jl/notebooks/Chapter09_Random_Walk.html

Reply:
oops, my bad

Reply:
the other error `MethodError: no method matching range(; start=-196, length=50, step=4)` is weird, couldn't reproduce offline